### DISCUSSION TITLE
Advanced Full Text Search

### DISCUSSION THREAD

**Discussion by lobosan** [directus/directus#17336] [Feature Requests]
Hi, I would love to have an API to build a full text search bar capable of indexing content from PDF and Office files. Also would be great to support filters based on taxonomies and language.

**Comment by rijkvanzanten** [comment]
Heya! Thank you for taking the time to submit this request! 

It has been over **90 days**, and this discussion has not received at least **15 votes** from the community. This means that we don't feel like there's enough community interest to warrant further R&D into this topic at this time. :ice_cube:

This request will now be closed to keep our discussions tidy. Please reach out if you have any questions!

_For more information, see our [Feature Request Process](https://docs.directus.io/contributing/feature-request-process.html)._
